.github/pull_request_template.md

```diff
@@ -1,4 +1,4 @@
-**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](https://github.com/spatialos/gdk-for-unity#give-us-feedback).
+**Contributions**: We are not currently taking public contributions - see our [contributions](../blob/master/.github/CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](https://github.com/spatialos/gdk-for-unity#give-us-feedback).
```

Comment:
link doesn't seem to work

Reply:
the link works when in actual PRs - which is what the template is for